do_snowDA.sh

```diff
@@ -63,8 +69,26 @@ THISDATE=${THISDATE:-"2015090118"}
 
 echo 'THISDATE in land DA, '$THISDATE
 
+############################################################################################
+if [ $ensemble_size -gt 1 ] && [ $do_ens == "YES" ]; then
+    declare -a ens_list=( $(for (( i=1; i<=${ensemble_size}; i++ )); do echo 0; done) ) # empty array
+    ensemble_count=0
+    for i in ${ens_list[@]}
+    do
+        ens_list[ensemble_count]="mem_`printf %02i $((ensemble_count + 1))`"
+        ensemble_count=$((ensemble_count+1))
+    done
+fi
+ens_list=(mem_pos mem_neg)
```

Comment:
This is overwriting the ens_list that you just constructed. We need to be able to run two possibilities: 

1. A true LETKF, using ensembles of model forecasts produced by your forcing. 
2. The LETKF-OI, which uses a pseudo ensemble created below by letkf_create_ens.py

I suggest that you change do_DA to have three options: 
"hofx" (that's the current "NO" - assume no ensemble for now), "LETKF-OI" and "LETKF". 

For "LETKF-OI" ens_list=(mem_pos mem_neg) 
for "LETKF" ens_list= the one you constructed from the ensemble number.

Then below, for "LETKF-OI" we'll need to call letkf_create_ens.py